Meta/Lagom/Tools/CodeGenerators/IPCCompiler/main.cpp

```diff
@@ -819,7 +821,11 @@ void build(StringBuilder& builder, Vector<Endpoint> const& endpoints)
 #if defined(AK_COMPILER_CLANG)
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wdefaulted-function-deleted"
-#endif)~~~");
+#else
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdefaulted-function-deleted"
+#endif
```

Comment:
Clang accepts `#pragma GCC` too, so the `#if defined(AK_COMPILER_CLANG)` part does not need to stay.